docs/source/example_use.rst

```diff
@@ -49,7 +49,7 @@ If you haven't followed the :doc:`getting started guide <getting_started>` to
 set up Orchestra yet, you should do that now. Also, make sure that
 ``'journalism_workflow'`` is in your ``INSTALLED_APPS`` setting, and that you
 have loaded the workflow into the database (``python manage.py loadworkflow
-journalism_workflow journalism_v1``).
+journalism_workflow v1``).
```

Comment:
I left this as `<APP_LABEL> <WORKFLOW_VERSION_SLUG>` intentionally to differentiate it from the `<WORKFLOW_SLUG>/WORKFLOW_VERSION_SLUG>` format accepted by the `loadworkflowsampledata` command. I'm open to suggestions for a cleaner way of highlighting this difference.